lib/galaxy/tools/xsd/galaxy.xsd

```diff
@@ -4888,9 +4888,9 @@ for more examples.
 ]]></xs:documentation>
     </xs:annotation>
     <xs:sequence/>
-    <xs:attribute name="input" type="xs:string" use="required">
+    <xs:attribute name="input" type="xs:string">
       <xs:annotation>
-        <xs:documentation xml:lang="en">This value must be the attribute name of
+        <xs:documentation xml:lang="en">This attribute should be the name of
 the desired input parameter (e.g. ``input="out_format"`` above).</xs:documentation>
```

Comment:
I think the documentation of `value` below is not correct, it says: `This value must also be an attribute name of an input parameter (e.g. ``value="interval"`` above).`, but should be: ```This must be a possible value of the ``input`` parameter (e.g. ``value="interval"`` above), or of the ``input_dataset``'s ``attribute.```